tests/gen_cert_test.py

```diff
@@ -181,3 +179,69 @@ def test_cert_date(self, force_date, locale, timezone, expected):
         """
         cert_date = get_cert_date(force_date, locale, timezone)
         assert_equal(expected, cert_date)
+
+
+class CertificateSubtemplateTests(unittest.TestCase):
+    """
+    Tests the subtemplates feature for certs
+
+    Subtemplates are currently written to key on designations only
+
+    These tests have a specific order
```

Comment:
Do these need to be 3 tests? Cause we're gonna have 1 subtemplate in the cert_data so all those for loops aren't actually needed...